Data/DataModel/Swag.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace ServerCore.DataModel
+{
+    public class Swag
+    {
+        public int ID { get; set; }
+
+        [Required]
+        public int EventId { get; set; }
+
+        public virtual Event Event { get; set; }
+
+        [Required]
+        public int PlayerId { get; set; }
+
+        public virtual PuzzleUser Player { get; set; }
+
+        public string Lunch { get; set; }
```

Comment:
Having this as a free form string seems like it could cause issues -- either performance or unexpected contents. I'd rather see a table of food options and have this as an identifier in that table.

Reply:
I was asked to have it be freeform to handle things like "no mayo", I have celiac", etc.